Modules/modExtendSaveAsText.ACM

```diff
@@ -85,21 +87,39 @@ Public Sub SaveAsText(objType As asObjectType, objName As String, strFilename As
   Select Case objType
     Case asObjectType.acForm, asObjectType.acReport
       Application.SaveAsText objType, objName, strFilename & ".tmp"
-      If IsUCS2(strFilename & ".tmp") Then
-        If CleanFormReportFile(strFilename & ".tmp", strFilename, Ignores) Then
-          If blnSaveAsUTF8 And IsUCS2(strFilename) Then
-            If Not ConvFileUCS2_to_UTF8(strFilename) Then 'convert file to UTF8
-              MsgBox "Unable to convert '" & objName & "' to UTF-8 encoding. Beware when committing."
-            End If
-          End If
-        Else
-          MsgBox "Something went wrong cleaning the file for " & objName & "."
+      blnUCS2 = IsUCS2(strFilename & ".tmp") ' indicates whether Application.SaveAsText produced UCS2 or ANSI output

```

Comment:
Can this not all be replaced with a call to `ConvFile_to_UCS2`?

Reply:
Agreed and implemented